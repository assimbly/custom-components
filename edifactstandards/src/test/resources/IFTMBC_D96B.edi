UNA:+.? 'UNB+UNOB:3+CARGOSMART:01+ICT+180705:0936+47850'UNH+4785000001+IFTMBC:D:96B:UN'BGM+770+2102838931+PE'DTM+137:20180705093438:204'TSR+30+3'FTX+AAI+++20?'Ctnr freistellen in Emmerich am 05.07, UN2735 KL8 II, Achtung Kumpo:rt buchen!'RFF+BN:2102838931'RFF+SC:FE140782v383'RFF+SI:169010'RFF+FF:807/USHY/55187'TDT+20+113South+1++OOLU:172:182+++9329526:ZZ1::OOCL KOBE'LOC+88+BEANR:229:6:DP World quai 1700+ANR:145:5:Antwerpen+BE:162:5:Belgium'DTM+63:201807122100:203'LOC+9+BEANR:139:6:Antwerp+ANR:145:5:Antwerp+BE:162:5:Belgium'DTM+133:20180713:102'LOC+11+TRIST:139:6:Istanbul (Ambarli)+IST:145:5:Istanbul (Ambarli)+TR:162:5:Turkey'DTM+132:20180731:102'LOC+7+TRIST:229:6:Kumport Liman Hizmetleri Lojistik Sanayi+IST:145:5:Istanbul+TR:162:5:Turkey'NAD+CA+OOLU:160'NAD+BK+3047402000:160++ICT Internationale Container Transp:ort GmbH+Sperberweg 6a+Neuss+NW+41468+DE'NAD+CZ+3047402000:160++ICT Internationale Container Transp:ort GmbH+Sperberweg 6a+Neuss+NW+41468+DE'NAD+FW+3047402000:160++ICT Internationale Container Transp:ort GmbH+Sperberweg 6a+Neuss+NW+41468+DE'GID+1+0:::5'FTX+AAA+++dangerous goods'GDS+11'MEA+WT+G+KGM:15588.000'DGS+IMD+8+2735++II+F-A, S'FTX+AAD+++Amines,C12-14-alkyldimethyl'FTX+AAD+++AMINES, LIQUID, CORROSIVE, N.O.S.'FTX+AAC+++OLBRE8930'CTA+HE+:KAO Chemicals GmbH'COM+34937399445:TE'EQD+CN+OOLU1025401+22G1+2++4'EQN+1'EQD+CN++22G1+++4'EQN+1'NAD+CK+RTM13++Rotterdam Container Terminal+Rotterdam Container Terminal Missou:riweg 17, Maasvlakte-Rotterdam 3199: LB'DTM+201:201806290800:203'EQD+CN+++++4'EQN+1'NAD+TR+ANR12++DP World quai 1700+Antwerp Gateway Terminal 9130 Kallo:-Doel Belgium'DTM+116:201807122100:203'UNT+42+4785000001'UNZ+1+47850'