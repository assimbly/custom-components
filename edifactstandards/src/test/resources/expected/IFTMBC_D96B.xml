<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
<Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blujaysolutions.com/2018/IFTMBC file:///C:/Users/jss/Documents/seacon/germany_interfaces/develop/iftmbc_D96B/IFTMBC_D96B.xsd" xmlns="http://www.blujaysolutions.com/2018/IFTMBC">
    <Interchange>
        <UNB>
            <F0020>47850</F0020>
            <S001>
                <F0001>UNOB</F0001>
                <F0002>3</F0002>
            </S001>
            <S002>
                <F0004>CARGOSMART</F0004>
                <F0007>01</F0007>
            </S002>
            <S003>
                <F0010>ICT</F0010>
            </S003>
            <S004>
                <F0017>180705</F0017>
                <F0019>936</F0019>
            </S004>
        </UNB>
        <UNZ>
            <F0036>1</F0036>
            <F0020>47850</F0020>
        </UNZ>
        <Group>
            <Message_IFTMBC>
                <UNH>
                    <F0062>4785000001</F0062>
                    <S009>
                        <F0065>IFTMBC</F0065>
                        <F0052>D</F0052>
                        <F0054>96B</F0054>
                        <F0051>UN</F0051>
                    </S009>
                </UNH>
                <BGM>
                    <F1225>PE</F1225>
                    <C002>
                        <F1001>770</F1001>
                    </C002>
                    <C106>
                        <F1004>2102838931</F1004>
                    </C106>
                </BGM>
                <DTM>
                    <C507>
                        <F2005>137</F2005>
                        <F2380>20180705093438</F2380>
                        <F2379>204</F2379>
                    </C507>
                </DTM>
                <TSR>
                    <C536>
                        <F4065>30</F4065>
                    </C536>
                    <C233>
                        <F7273_1>3</F7273_1>
                    </C233>
                </TSR>
                <FTX>
                    <F4451>AAI</F4451>
                    <C108>
                        <F4440>20'Ctnr freistellen in Emmerich am 05.07, UN2735 KL8 II, Achtung Kumpo</F4440>
                        <F4440>rt buchen!</F4440>
                    </C108>
                </FTX>
                <UNT>
                    <F0074>42</F0074>
                    <F0062>4785000001</F0062>
                </UNT>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>BN</F1153>
                            <F1154>2102838931</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>SC</F1153>
                            <F1154>FE140782v383</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>SI</F1153>
                            <F1154>169010</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>FF</F1153>
                            <F1154>807/USHY/55187</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG3>
                    <TDT>
                        <F8051>20</F8051>
                        <F8028>113South</F8028>
                        <C220>
                            <F8067>1</F8067>
                        </C220>
                        <C040>
                            <F3127>OOLU</F3127>
                            <F1131>172</F1131>
                            <F3055>182</F3055>
                        </C040>
                        <C222>
                            <F8213>9329526</F8213>
                            <F1131>ZZ1</F1131>
                            <F8212>OOCL KOBE</F8212>
                        </C222>
                    </TDT>
                    <SG4>
                        <LOC>
                            <F3227>88</F3227>
                            <C517>
                                <F3225>BEANR</F3225>
                                <F1131>229</F1131>
                                <F3055>6</F3055>
                                <F3224>DP World quai 1700</F3224>
                            </C517>
                            <C519>
                                <F3223>ANR</F3223>
                                <F1131>145</F1131>
                                <F3055>5</F3055>
                                <F3222>Antwerpen</F3222>
                            </C519>
                            <C553>
                                <F3233>BE</F3233>
                                <F1131>162</F1131>
                                <F3055>5</F3055>
                                <F3232>Belgium</F3232>
                            </C553>
                        </LOC>
                        <DTM>
                            <C507>
                                <F2005>63</F2005>
                                <F2380>201807122100</F2380>
                                <F2379>203</F2379>
                            </C507>
                        </DTM>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>9</F3227>
                            <C517>
                                <F3225>BEANR</F3225>
                                <F1131>139</F1131>
                                <F3055>6</F3055>
                                <F3224>Antwerp</F3224>
                            </C517>
                            <C519>
                                <F3223>ANR</F3223>
                                <F1131>145</F1131>
                                <F3055>5</F3055>
                                <F3222>Antwerp</F3222>
                            </C519>
                            <C553>
                                <F3233>BE</F3233>
                                <F1131>162</F1131>
                                <F3055>5</F3055>
                                <F3232>Belgium</F3232>
                            </C553>
                        </LOC>
                        <DTM>
                            <C507>
                                <F2005>133</F2005>
                                <F2380>20180713</F2380>
                                <F2379>102</F2379>
                            </C507>
                        </DTM>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>11</F3227>
                            <C517>
                                <F3225>TRIST</F3225>
                                <F1131>139</F1131>
                                <F3055>6</F3055>
                                <F3224>Istanbul (Ambarli)</F3224>
                            </C517>
                            <C519>
                                <F3223>IST</F3223>
                                <F1131>145</F1131>
                                <F3055>5</F3055>
                                <F3222>Istanbul (Ambarli)</F3222>
                            </C519>
                            <C553>
                                <F3233>TR</F3233>
                                <F1131>162</F1131>
                                <F3055>5</F3055>
                                <F3232>Turkey</F3232>
                            </C553>
                        </LOC>
                        <DTM>
                            <C507>
                                <F2005>132</F2005>
                                <F2380>20180731</F2380>
                                <F2379>102</F2379>
                            </C507>
                        </DTM>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>7</F3227>
                            <C517>
                                <F3225>TRIST</F3225>
                                <F1131>229</F1131>
                                <F3055>6</F3055>
                                <F3224>Kumport Liman Hizmetleri Lojistik Sanayi</F3224>
                            </C517>
                            <C519>
                                <F3223>IST</F3223>
                                <F1131>145</F1131>
                                <F3055>5</F3055>
                                <F3222>Istanbul</F3222>
                            </C519>
                            <C553>
                                <F3233>TR</F3233>
                                <F1131>162</F1131>
                                <F3055>5</F3055>
                                <F3232>Turkey</F3232>
                            </C553>
                        </LOC>
                    </SG4>
                </SG3>
                <SG6>
                    <NAD>
                        <F3035>CA</F3035>
                        <C082>
                            <F3039>OOLU</F3039>
                            <F1131>160</F1131>
                        </C082>
                    </NAD>
                </SG6>
                <SG6>
                    <NAD>
                        <F3035>BK</F3035>
                        <F3164>Neuss</F3164>
                        <F3229>NW</F3229>
                        <F3251>41468</F3251>
                        <F3207>DE</F3207>
                        <C082>
                            <F3039>3047402000</F3039>
                            <F1131>160</F1131>
                        </C082>
                        <C080>
                            <F3036>ICT Internationale Container Transp</F3036>
                            <F3036>ort GmbH</F3036>
                        </C080>
                        <C059>
                            <F3042>Sperberweg 6a</F3042>
                        </C059>
                    </NAD>
                </SG6>
                <SG6>
                    <NAD>
                        <F3035>CZ</F3035>
                        <F3164>Neuss</F3164>
                        <F3229>NW</F3229>
                        <F3251>41468</F3251>
                        <F3207>DE</F3207>
                        <C082>
                            <F3039>3047402000</F3039>
                            <F1131>160</F1131>
                        </C082>
                        <C080>
                            <F3036>ICT Internationale Container Transp</F3036>
                            <F3036>ort GmbH</F3036>
                        </C080>
                        <C059>
                            <F3042>Sperberweg 6a</F3042>
                        </C059>
                    </NAD>
                </SG6>
                <SG6>
                    <NAD>
                        <F3035>FW</F3035>
                        <F3164>Neuss</F3164>
                        <F3229>NW</F3229>
                        <F3251>41468</F3251>
                        <F3207>DE</F3207>
                        <C082>
                            <F3039>3047402000</F3039>
                            <F1131>160</F1131>
                        </C082>
                        <C080>
                            <F3036>ICT Internationale Container Transp</F3036>
                            <F3036>ort GmbH</F3036>
                        </C080>
                        <C059>
                            <F3042>Sperberweg 6a</F3042>
                        </C059>
                    </NAD>
                </SG6>
                <SG9>
                    <GID>
                        <F1496>1</F1496>
                        <C213>
                            <F7224>0</F7224>
                            <F3055>5</F3055>
                        </C213>
                    </GID>
                    <FTX>
                        <F4451>AAA</F4451>
                        <C108>
                            <F4440>dangerous goods</F4440>
                        </C108>
                    </FTX>
                    <GDS>
                        <C703>
                            <F7085>11</F7085>
                        </C703>
                    </GDS>
                    <SG11>
                        <MEA>
                            <F6311>WT</F6311>
                            <C502>
                                <F6313>G</F6313>
                            </C502>
                            <C174>
                                <F6411>KGM</F6411>
                                <F6314>15588.000</F6314>
                            </C174>
                        </MEA>
                    </SG11>
                    <SG15>
                        <DGS>
                            <F8273>IMD</F8273>
                            <F8339>II</F8339>
                            <F8364>F-A, S</F8364>
                            <C205>
                                <F8351>8</F8351>
                            </C205>
                            <C234>
                                <F7124>2735</F7124>
                            </C234>
                        </DGS>
                        <FTX>
                            <F4451>AAD</F4451>
                            <C108>
                                <F4440>Amines,C12-14-alkyldimethyl</F4440>
                            </C108>
                        </FTX>
                        <FTX>
                            <F4451>AAD</F4451>
                            <C108>
                                <F4440>AMINES, LIQUID, CORROSIVE, N.O.S.</F4440>
                            </C108>
                        </FTX>
                        <FTX>
                            <F4451>AAC</F4451>
                            <C108>
                                <F4440>OLBRE8930</F4440>
                            </C108>
                        </FTX>
                        <SG16>
                            <CTA>
                                <F3139>HE</F3139>
                                <C056>
                                    <F3412>KAO Chemicals GmbH</F3412>
                                </C056>
                            </CTA>
                            <COM>
                                <C076>
                                    <F3148>34937399445</F3148>
                                    <F3155>TE</F3155>
                                </C076>
                            </COM>
                        </SG16>
                    </SG15>
                </SG9>
                <SG18>
                    <EQD>
                        <F8053>CN</F8053>
                        <F8077>2</F8077>
                        <F8169>4</F8169>
                        <C237>
                            <F8260>OOLU1025401</F8260>
                        </C237>
                        <C224>
                            <F8155>22G1</F8155>
                        </C224>
                    </EQD>
                    <EQN>
                        <C523>
                            <F6350>1</F6350>
                        </C523>
                    </EQN>
                </SG18>
                <SG18>
                    <EQD>
                        <F8053>CN</F8053>
                        <F8169>4</F8169>
                        <C224>
                            <F8155>22G1</F8155>
                        </C224>
                    </EQD>
                    <EQN>
                        <C523>
                            <F6350>1</F6350>
                        </C523>
                    </EQN>
                    <SG19>
                        <NAD>
                            <F3035>CK</F3035>
                            <C082>
                                <F3039>RTM13</F3039>
                            </C082>
                            <C080>
                                <F3036>Rotterdam Container Terminal</F3036>
                            </C080>
                            <C059>
                                <F3042>Rotterdam Container Terminal Missou</F3042>
                                <F3042>riweg 17, Maasvlakte-Rotterdam 3199</F3042>
                                <F3042> LB</F3042>
                            </C059>
                        </NAD>
                        <DTM>
                            <C507>
                                <F2005>201</F2005>
                                <F2380>201806290800</F2380>
                                <F2379>203</F2379>
                            </C507>
                        </DTM>
                    </SG19>
                </SG18>
                <SG18>
                    <EQD>
                        <F8053>CN</F8053>
                        <F8169>4</F8169>
                    </EQD>
                    <EQN>
                        <C523>
                            <F6350>1</F6350>
                        </C523>
                    </EQN>
                    <SG19>
                        <NAD>
                            <F3035>TR</F3035>
                            <C082>
                                <F3039>ANR12</F3039>
                            </C082>
                            <C080>
                                <F3036>DP World quai 1700</F3036>
                            </C080>
                            <C059>
                                <F3042>Antwerp Gateway Terminal 9130 Kallo</F3042>
                                <F3042>-Doel Belgium</F3042>
                            </C059>
                        </NAD>
                        <DTM>
                            <C507>
                                <F2005>116</F2005>
                                <F2380>201807122100</F2380>
                                <F2379>203</F2379>
                            </C507>
                        </DTM>
                    </SG19>
                </SG18>
            </Message_IFTMBC>
        </Group>
    </Interchange>
</Envelope>